build.js

```diff
@@ -0,0 +1,34 @@
+let esbuild = require('esbuild');
+const path = require('path');
+
+let entries = {
+  liveDemoBox: './templates/static/js/src/demobox/app.tsx',
+};
+
+const isDev = process && process.env && process.env.NODE_ENV === 'development';
+
+for (const [key, value] of Object.entries(entries)) {
+  const options = {
+    entryPoints: [value],
+    bundle: true,
+    minify: isDev ? false : true,
+    nodePaths: [path.resolve(__dirname, './templates/static/js/src')],
+    sourcemap: isDev ? false : true,
+    outfile: 'templates/static/js/dist/' + key + '.js',
```

Comment:
We currently build SCSS files to build/css directory (which is linked into static folder as `templates/static/build/css`) so for consistency maybe we can build JS into build/js, and it  should already be visible in static as `templates/static/build/js`). So we would avoid adding another "dist" folder and ignoring it.

Reply:
I changed the path to `templates/static/build/js`.